### PR TITLE
Fix PreferencesClient to use non-namespaced URLs

### PR DESCRIPTION
**Pull request by jwang47**

### CODE REVIEW

**Comment by jwang47** [comment]
Build running http://builds.cask.co/browse/CDAP-DUT811-1

File: cdap-client/src/main/java/co/cask/cdap/client/config/ClientConfig.java [hunk 1]
```diff
@@ -100,7 +125,7 @@ public URL resolveURL(String path) throws MalformedURLException {
    * @return URL of the resolved path
    * @throws MalformedURLException
    */
-  public URL resolveURL(String apiVersion, String namespace, String path) throws MalformedURLException {
+  public URL resolveNamespacedURL(String apiVersion, String namespace, String path) throws MalformedURLException {
```
Comment (anwar6953) [inline cdap-client/src/main/java/co/cask/cdap/client/config/ClientConfig.java]: do we need this method?

Comment (bdmogal) [inline cdap-client/src/main/java/co/cask/cdap/client/config/ClientConfig.java]: Shouldn't need  ``apiVersion`` for namespaced URLs, no? Should always be v3?

**Comment by jwang47** [comment]
Build running http://builds.cask.co/browse/CDAP-DUT811-2

**Comment by anwar6953** [comment]
LGTM

**Pull request merged**